numpy/lib/function_base.py

```diff
@@ -33,7 +33,6 @@
 import builtins
 
 # needed in this module for compatibility
-from numpy.lib.histograms import histogram, histogramdd
```

Comment:
idem., See the comment on the line above.

Reply:
Should these be added to `__all__` then?

Reply:
Might be worthwhile to figure out why exactly they're here in the first place.
Based on the answer deprecating them might be desirable over adding them to `__all__`.

Reply:
Ah, found it: four years ago they were moved from `function_base` to `histograms` in a namespace clean up (https://github.com/numpy/numpy/pull/10186). 
Honestly, this period is long enough that I'd advocate for their deprecation.